installer/src/orbic.rs

```diff
@@ -0,0 +1,136 @@
+use std::io::Write;
+use std::path::Path;
+use std::time::Duration;
+
+use adb_client::{ADBDeviceExt, ADBUSBDevice};
+use anyhow::{Result, anyhow};
+use nusb::Interface;
+
+use crate::{RELEASE, serial};
+
+const ORBIC_NOT_FOUND: &str = r#"No Orbic device found.
+Make sure your device is plugged in and turned on.
+
+If it's possible you have a device with a different usb id:
+please file a bug with the output of `lsusb` attached."#;
```

Comment:
might reword this to be a little more user-friendly to less technical users:

```suggestion
If you're sure you've plugged in an Orbic device via USB, there may be a bug in
our installer. Please file a bug with the output of `lsusb` attached.
```